stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java

```diff
@@ -202,11 +220,87 @@ public synchronized void mergeThreadResult(MasterBenchTaskResult threadResult) {
       }
     }
 
+    @SuppressFBWarnings(value = "DMI_HARDCODED_ABSOLUTE_FILENAME")
+    public synchronized void addAdditionalResult() throws IOException {
+      if (mResult == null) {
+        return;
+      }
+
+      Map<String, PartialResultStatistic> methodNameToHistogram = new HashMap<>();
```

Comment:
I think this parsing should be pulled out into a library, since I think it will be reused in the future.

Reply:
I agree that it will. But I am not sure how it will be used in the future so leaving it in here for now. Refactoring this out will be easy and will go to a more appropriate place then.